### PR TITLE
MHP-2807-Fix-The-Unassigned-Count

### PR DESCRIPTION
**Pull request by TheNoodleMoose**
If a person is added or deleted I just emit _updateFilterCount_ and catch that emit in the peopleFiltersPanel component. If there is a better way for catching if an update occurs let me know.

### CODE REVIEW

File: app/assets/javascripts/angular/components/peopleFiltersPanel/peopleFiltersPanel.component.js [hunk 1]
```diff
@@ -32,8 +32,6 @@ function peopleFiltersPanelController(
     this.filters = {};
 
     this.$onInit = () => {
-        loadFilterStats();
```
Comment (OzzieOrca) [inline app/assets/javascripts/angular/components/peopleFiltersPanel/peopleFiltersPanel.component.js]: Do we still want this to run initially? Or does something else handle it?

File: app/assets/javascripts/angular/components/peopleFiltersPanel/peopleFiltersPanel.component.js [hunk 2]
```diff
@@ -106,6 +108,8 @@ function peopleFiltersPanelController(
             };
         }
 
+        loadFilterStats();
```
Comment (OzzieOrca) [inline app/assets/javascripts/angular/components/peopleFiltersPanel/peopleFiltersPanel.component.js]: Why do we need to do this here?

File: app/assets/javascripts/angular/components/peopleFiltersPanel/peopleFiltersPanel.component.js (hunk 1 above)
Comment (TheNoodleMoose) [reply app/assets/javascripts/angular/components/peopleFiltersPanel/peopleFiltersPanel.component.js]: Yeah, I will move it back into the init. It was running due to the other thing you commented on. It makes more sense to run it in the init.

File: app/assets/javascripts/angular/components/peopleFiltersPanel/peopleFiltersPanel.component.js (hunk 2 above)
Comment (TheNoodleMoose) [reply app/assets/javascripts/angular/components/peopleFiltersPanel/peopleFiltersPanel.component.js]: Yeah somewhere along the way it got moved here before I added the $rootScope catch for testing. I will move it from here back into the init.

**Pull request merged**